index.html

```diff
@@ -74,17 +76,17 @@
 <pre>
 <center><b>Image Scrubber</b></center>
 
-This is a tool for anonymizing photographs taken at protests. 
+This is a tool for anonymizing photographs taken at protests.
 
-It will remove identifying metadata (<a href="https://en.wikipedia.org/wiki/Exif">Exif data</a>) from photographs, and also allow you to selectively blur parts of the image to cover faces and other identifiable information. 
+It will remove identifying metadata (<a href="https://en.wikipedia.org/wiki/Exif">Exif data</a>) from photographs, and also allow you to selectively blur parts of the image to cover faces and other identifiable information.
 
 Hit the open button to open a photograph. The program will display the data it is removing.
```

Comment:
Nit: it would be good to briefly mention the drag-and-drop functionality in the instructions here too.